src/App.scss

```diff
@@ -1,19 +1,18 @@
 html {
-    background: #ddd;  
+    background: #ddd;
     height: 100%;
-    display: flex;
   }
   body {
     width: 100%;
-    margin: auto;
+    display: flex;
+    justify-content: center;
   }
-  
+
   .App {
     font-family: sans-serif;
     text-align: center;
-    padding: 550px;
   }
```

Comment:
solved a minor styling issue, it wasn't centered in the screen:


https://github.com/user-attachments/assets/473b296e-124e-47c2-8197-9cdf40842b4a